app/view/show.py

```diff
@@ -244,6 +252,8 @@ def show_group(avg):
 
     group = data['group']
     advisories = data['advisories']
+    if not current_user.role.is_reporter:
+        advisories = list(filter(lambda advisory: advisory.publication == Publication.published, advisories))
```

Comment:
I wonder if we could take the 3 instances of this if (and filter) to a separate function for readability:

```python
advisories = _fetch_and_filter_advisories(data['advisories'], current_user.role)
```